src/wallet/rpc/transactions.cpp

```diff
@@ -431,6 +432,7 @@ static const std::vector<RPCResult> TransactionDescriptionString()
            {RPCResult::Type::NUM, "blockindex", /*optional=*/true, "The index of the transaction in the block that includes it."},
            {RPCResult::Type::NUM_TIME, "blocktime", /*optional=*/true, "The block time expressed in " + UNIX_EPOCH_TIME + "."},
            {RPCResult::Type::STR_HEX, "txid", "The transaction id."},
+           {RPCResult::Type::STR_HEX, "wtxid", "The witness transaction id."},
```

Comment:
nit: The two existing "wtxid" descriptions are different.

I prefer `"hash of serialized transaction, including witness data"`

Reply:
I prefer `"hash of serialized transaction, including witness data"` as well, going to change it.

Reply:
# []()